examples/step-68/doc/intro.dox

```diff
@@ -0,0 +1,149 @@
+
+Generating the particles is not straightforward since the processor to which they belong
+must first be identified before the cell in which they are located is found.
+Deal.II provides numerous capabilities to generate particles through the Particles::Generator namespace.
+Some of these particle generator generate particles on the locally own subdomain. For example,
+Particles::Generators::regular_reference_locations uses a regular reference location within each cell of
+the subdomain or Particles::Generators::probabilistic_locations uses
+a probability density function to generate the particles.
+
+In some situations, such as the present step, particles must be generated at specific locations
+on cells that are owned only by a subset of the processors. In  most of these situations,
+the insertion of the particle is done for a very limited number of time-steps and, consequently,
+does not constitute a large portion of the computational cost. For these occasions, deal.II provides
+convenient Particles::Generators that can globally insert the particles even if they are not located
+on the subdomain from which they are created. The generators first locate on which subdomain the particles
+are situated, identify is which cell they are located and exchange the necessary information amongst the processors
+to ensure that the particle is generated with the right properties. Consequently, this type of particle generation
+is communication intensive. The Particles::Generators::dof_support_points and the Particles::Generators::quadrature_points
+generate particles using a, possible non-matching, triangulation and the points of an associated dof_handler or quadrature respectively.
+Furthermore, the Particles::ParticleHandler class provides the Particles::ParticleHandler::insert_global_particles function
+which enables the global insertion of particles from a vector of points and a global vector of bounding boxes. In the present step,
+we use the Particles::Generators::quadrature_points to insert the particle in the shape of a circle.
+
+
+<h3>The testcase</h3>
+
+In the present step, we use particles as massless tracers to illustrate
+the dynamics of a particular vortical flow : the Rayleigh-Kotte Vortex. This flow pattern
+is generally used as a complex test case for interface tracking methods
+(e.g. volume-of-fluid and level set approches) since
+it leads to strong rotation and elongation of the fluid (Blais 2013).
+
+The stream function $\Psi$ of this Rayleigh-Kotte vortex is defined as:
+
+@f[
+\Psi = \frac{1}{\pi} sin^2 (\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f]
+where $T$ is the period of the flow. The velocity profile in 2D ($\textbf{u}=[u,v]^T$) is :
+@f{eqnarray*}
+   u &=&  - \frac{\partial\Psi}{\partial y} = -2 \sin^2 (\pi x) \sin (\pi y) \cos (\pi y)  \cos \left( \pi \frac{t}{T} \right)\\
+   v &=&  \frac{\partial\Psi}{\partial x} = 2 \cos(\pi x) \sin(\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f}
+
+
```

Comment:
Maybe add a sentence like the following explaining that the flow will reverse periodically and material will end up where it started:

```
It can be seen that this velocity reverses periodically due to the term $\cos \left( \pi \frac{t}{T} \right)$ and that material will end up at its starting position after every period of length $t=2T$. We will run this tutorial program for exactly one period and compare the final particle location to the initial location to illustrate this flow property.